src/model/Post.js

```diff
@@ -0,0 +1,102 @@
+import {EventEmitter} from "events";
+
+class Post extends EventEmitter{
```

Comment:
PostModel would have been a better name :) Usually you would name the entity itself `Post` and the model which handles this entity `PostModel`.